### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
Looks good to me.

**Review by Debarshi95 — APPROVED** [review]

File: Product.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    width: 1.2rem;
+    height: 1.2rem;
+    background-color: var(--bg-card-button-color1);
+    border-radius: 50%;
+    border: none;
+  }
+  .Wishlist-container .card2-bottom-section {
+    padding-left: 1rem;
+  }
+  ins {
```
Comment (ashishkatiyar2404) [inline Product.css]: try to use classes always for styling.

File: Product.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    padding-left: 5px;
+    border-bottom: 1px solid var(--divider-bg-color);
+    position: relative;
+    border-right: 1px solid var(--divider-bg-color);
+}
+.left-sidebar-filter-section ul{
+  margin: 0px;
+}
+.category-container
+{
```
Comment (ashishkatiyar2404) [inline Product.css]:
```suggestion
.category-container {

```

File: Product.css [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+}
+.header-container
+{
+    position: relative;
+    padding-top: 0;   
+}
+.header-title
+{
+    font-weight: 700;
+}
```
Comment (ashishkatiyar2404) [inline Product.css]:
```suggestion
.header-title {
    font-weight: 700;
}
```

**Review by ashishkatiyar2404 — APPROVED** [review]

**Comment by ashishkatiyar2404** [comment]
your code looks good to merge.
A small suggestion pls give space between two classes in  .css files (for increase Readability)

File: Product.css (hunk 2 above)
Comment (ShrutiLakhani) [reply Product.css]: Done

**Pull request merged**